m4_sessions/1711-inning/Group_4.md

```diff
@@ -1,9 +1,25 @@
 Group Four
 
-Session Title: Description: Why should students come to this session?
+Session Title: How to write a kicka$$ technical blog post
+
+Description: Why should students come to this session?
+
+  * A technical blog will highlight students programming strengths and will convey what they currently know.
+  * It also serves as good self-marketing (which can lead to a job!)
 
 Session Outcomes: What will students leave with or gain from this session?
 
+  * They will gain the practical skillset to write their first technical article.
+
 Session Outline: Include the flow of the session and the time for each section.
 
+  * WHY - why it's important to write a technical blog post
+  * WHAT - topic of the blog
+  * HOW - how to write a technical blog (provide them a list of guidelines that will allow them to start writing that day)
+  * WHEN & WHERE - When and where to promote your blog.
```

Comment:
We have had people get jobs from blogs.  I also like that for a title. "Jobs from blogs" haha!  Okay, but this is possible and pushing them to actually publish will be key.  (More than one article!)